tests/docs/test_creating_spark_session.py

```diff
@@ -0,0 +1,19 @@
+"""Testing creating spark session docs."""
+from gentropy.common.session import Session
+
+from docs.src_snippets.howto.python_api.creating_spark_session import (
+    custom_session,
+    default_session,
+)
+
+
+def test_default_session() -> None:
+    """Test default session."""
+    session = default_session()
+    assert isinstance(session, Session)
+
+
+def test_custom_session() -> None:
```

Comment:
I tried an assert to evaluate that the session is using 4G and it isn't. I think `session` contains the pytest session (probably through `getOrCreate`).

I'm not too concern about this, just saying is not an extremely canonical test, but it should be able to pick if something changes with the Session class.

Reply:
So the code inside the how-tos are filled from an actual python scripts that are tested. Great.